lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntimeDynamicTypeResolution.cpp

```diff
@@ -664,17 +664,21 @@ SwiftLanguageRuntimeImpl::emplaceClangTypeInfo(
     m_clang_type_info.insert({clang_type.GetOpaqueQualType(), llvm::None});
     return nullptr;
   }
+  auto byte_align = *bit_align / 8;
```

Comment:
is `bit_align` guaranteed to be a multiple of 8?

Reply:
Not sure, I just moved that from a block of code below:

```
         swift::reflection::TypeInfo(swift::reflection::TypeInfoKind::Builtin,
                                      *byte_size, *bit_align / 8, 0, 0, true)});
```

I'll find out.

Reply:
Do you think we have an alignment that's not expressible in bytes though?

Reply:
I wouldn't think so. Sounds like this calls for an assert.